package/python-kiwi-spec-template

```diff
@@ -295,7 +295,9 @@ Summary:        KIWI - Host system dependencies
 Group:          %{sysgroup}
 Requires:       kiwi-systemdeps-core = %{version}-%{release}
 Requires:       kiwi-systemdeps-bootloaders = %{version}-%{release}
-%if 0%{?fedora} || 0%{?rhel} || 0%{?suse_version}
+%if 0%{?fedora} || 0%{?rhel} || 0%{?sle_version} >= 150000 && !0%{?is_opensuse} 
```

Comment:
For the rhel part this should be ```0%{?rhel} >= 8```

I remember Neal saying that recommends only works starting with rhel >=8